browser/brave_wallet/brave_wallet_service_delegate_impl_unittest.cc

```diff
@@ -33,23 +33,23 @@ const char* valid_data =
     "\\\"salt\\\":\\\"z1bTZtBY33d2l6CfiFs5V/eRQLS6Qsq5UtAQOIfaIps=\\\"}\"}}}";
 }  // namespace
 
-class BraveWalletImporterDelegateImplUnitTest : public testing::Test {
+class BraveWalletServiceDelegateImplUnitTest : public testing::Test {
  public:
-  BraveWalletImporterDelegateImplUnitTest() = default;
+  BraveWalletServiceDelegateImplUnitTest() = default;
 
   void TearDown() override { importer_.reset(); }
 
   void SetUp() override {
     keyring_controller_ =
         KeyringControllerFactory::GetControllerForContext(browser_context());
     importer_ =
```

Comment:
s/importer_/delegate_/g

Reply:
Fixed and rebased in 09f4690d47407505433a9fc290366a321c594682